### PR TITLE
fix: upgrade everything so it works on mac

### LINKED ISSUE #90: Installation is broken on Mac (BigSur 1.1 20C69)
![image](https://user-images.githubusercontent.com/1283200/104850132-08414500-58cc-11eb-99e9-5bcbfa236dce.png)


<details>
<summary> Full error </summary>

```
   info building revery@github:revery-ui/revery#ac848c4@d41d8cd9                                                                                                                                                                        [165/613]
error: build failed with exit code: 1
  build log:
    # esy-build-package: building: revery@github:revery-ui/revery#ac848c4
    # esy-build-package: pwd: /Users/fakenickels/.esy/source/i/revery__5c5d55b8
    # esy-build-package: running: 'dune' 'build' '-p' 'reason-harfbuzz,reason-skia,reason-sdl2,Revery'
        discover packages/reason-sdl2/src/c_flags.sexp,packages/reason-sdl2/src/c_library_flags.sexp,packages/reason-sdl2/src/cxx_flags.sexp,packages/reason-sdl2/src/flags.sexp
    SDL2 Library Folder Path: /Users/fakenickels/.esy/3_____________________________________________________________/i/esy_sdl2-2.0.10008-de4f481b/lib
             gcc packages/reason-skia/src/wrapped/c/c_stubs.o
    c_stubs.c:30:53: warning: incompatible pointer types passing 'void *(void *, const char *)' to parameter of type 'gr_gl_get_proc' (aka 'void (*(*)(void *, const char *))(void)') [-Wincompatible-pointer-types]
                gr_glinterface_assemble_gl_interface(0, reason_skia_sdl2_get);
                                                        ^~~~~~~~~~~~~~~~~~~~
    /Users/fakenickels/.esy/3_____________________________________________________________/i/esy_skia-d35022e2/include/c/gr_context.h:40:97: note: passing argument to parameter 'get' here
    SK_C_API const gr_glinterface_t* gr_glinterface_assemble_gl_interface(void* ctx, gr_gl_get_proc get);
                                                                                                    ^
    c_stubs.c:29:23: warning: initializing 'gr_glinterface_t *' (aka 'struct gr_glinterface_t *') with an expression of type 'const gr_glinterface_t *' (aka 'const struct gr_glinterface_t *') discards qualifiers [-Wincompatible-pointer-ty
pes-discards-qualifiers]
        gr_glinterface_t *interface =
                          ^
    c_stubs.c:36:55: warning: incompatible pointer types passing 'void *(void *, const char *)' to parameter of type 'gr_gl_get_proc' (aka 'void (*(*)(void *, const char *))(void)') [-Wincompatible-pointer-types]
                gr_glinterface_assemble_gles_interface(0, reason_skia_sdl2_get);
                                                          ^~~~~~~~~~~~~~~~~~~~
    /Users/fakenickels/.esy/3_____________________________________________________________/i/esy_skia-d35022e2/include/c/gr_context.h:41:99: note: passing argument to parameter 'get' here
    SK_C_API const gr_glinterface_t* gr_glinterface_assemble_gles_interface(void* ctx, gr_gl_get_proc get);
                                                                                                      ^
    c_stubs.c:35:23: warning: initializing 'gr_glinterface_t *' (aka 'struct gr_glinterface_t *') with an expression of type 'const gr_glinterface_t *' (aka 'const struct gr_glinterface_t *') discards qualifiers [-Wincompatible-pointer-ty
pes-discards-qualifiers]                                                                                                                                                                                                                              gr_glinterface_t *interface =
                          ^
    4 warnings generated.
          ocamlc src/Native/ReveryAppDelegate.o (exit 2)
    (cd /Users/fakenickels/.esy/3/b/revery-bd600809/default/src/Native && /Users/fakenickels/.esy/3_____________________________________________________________/i/ocaml-4.9.1000-c0c8387c/bin/ocamlc.opt -g -I /Users/fakenickels/.esy/3_____
________________________________________________________/i/opam__s__lru-e58036fa/lib/lru -I /Users/fakenickels/.esy/3_____________________________________________________________/i/opam__s__lwt-opam__c__4.5.0-9cfc7df4/lib/lwt -I /Users/fa
kenickels/.esy/3_____________________________________________________________/i/opam__s__ocamlfind-opam__c__1.8.1-ad96dfd8/lib/bytes -I /Users/fakenickels/.esy/3_____________________________________________________________/i/opam__s__psq-
opam__c__0.2.0-d409d938/lib/psq -I /Users/fakenickels/.esy/3_____________________________________________________________/i/opam__s__result-opam__c__1.5-ce40ece0/lib/result -I /Users/fakenickels/.esy/3_____________________________________
________________________/i/opam__s__seq-opam__c__base-15764575/lib/seq -I ../../packages/reason-sdl2/src -ccopt -O2 -ccopt -fno-strict-aliasing -ccopt -fwrapv -ccopt -Wall -ccopt -Wextra -ccopt -Werror -ccopt -I -ccopt . -ccopt -x -ccopt
objective-c -ccopt -g -o ReveryAppDelegate.o ReveryAppDelegate.c)                                                                                                                                                                                 ReveryAppDelegate.c:48:33: error: 'NSUserNotificationCenter' is deprecated: first deprecated in macOS 11.0 - All NSUserNotifications API should be replaced with UserNotifications.frameworks API [-Werror,-Wdeprecated-declarations]
    - (void)userNotificationCenter:(NSUserNotificationCenter *)center
                                    ^
    /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks/Foundation.framework/Headers/NSUserNotification.h:118:12: note: 'NSUserNotificationCenter' has been explicitly marked deprecated here
    @interface NSUserNotificationCenter : NSObject {
               ^
    ReveryAppDelegate.c:49:30: error: 'NSUserNotification' is deprecated: first deprecated in macOS 11.0 - All NSUserNotifications API should be replaced with UserNotifications.frameworks API [-Werror,-Wdeprecated-declarations]
        didActivateNotification:(NSUserNotification *)notification {
                                 ^
    /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks/Foundation.framework/Headers/NSUserNotification.h:24:12: note: 'NSUserNotification' has been explicitly marked de
precated here
    @interface NSUserNotification : NSObject <NSCopying> {
               ^
    ReveryAppDelegate.c:59:33: error: 'NSUserNotificationCenter' is deprecated: first deprecated in macOS 11.0 - All NSUserNotifications API should be replaced with UserNotifications.frameworks API [-Werror,-Wdeprecated-declarations]
    - (BOOL)userNotificationCenter:(NSUserNotificationCenter *)center
                                    ^
    /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks/Foundation.framework/Headers/NSUserNotification.h:118:12: note: 'NSUserNotificationCenter' has been explicitly ma
rked deprecated here
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks/Foundation.framework/Headers/NSUserNotification.h:118:12: note: 'NSUserNotificationCenter' has been expl[111/613]
rked deprecated here
    @interface NSUserNotificationCenter : NSObject {
               ^
    ReveryAppDelegate.c:60:32: error: 'NSUserNotification' is deprecated: first deprecated in macOS 11.0 - All NSUserNotifications API should be replaced with UserNotifications.frameworks API [-Werror,-Wdeprecated-declarations]
        shouldPresentNotification:(NSUserNotification *)notification {
                                   ^
    /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks/Foundation.framework/Headers/NSUserNotification.h:24:12: note: 'NSUserNotification' has been explicitly marked de
precated here
    @interface NSUserNotification : NSObject <NSCopying> {
               ^
    ReveryAppDelegate.c:42:32: error: 'NSUserNotificationCenter' is deprecated: first deprecated in macOS 11.0 - All NSUserNotifications API should be replaced with UserNotifications.frameworks API [-Werror,-Wdeprecated-declarations]
        [[NSUserNotificationCenter defaultUserNotificationCenter] setDelegate:self];
                                   ^
    /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks/Foundation.framework/Headers/NSUserNotification.h:118:12: note: 'NSUserNotificationCenter' has been explicitly ma
rked deprecated here
    @interface NSUserNotificationCenter : NSObject {
               ^
    5 errors generated.
          ocamlc src/Native/notification_cocoa.o (exit 2)
    (cd /Users/fakenickels/.esy/3/b/revery-bd600809/default/src/Native && /Users/fakenickels/.esy/3_____________________________________________________________/i/ocaml-4.9.1000-c0c8387c/bin/ocamlc.opt -g -I /Users/fakenickels/.esy/3_____
________________________________________________________/i/opam__s__lru-e58036fa/lib/lru -I /Users/fakenickels/.esy/3_____________________________________________________________/i/opam__s__lwt-opam__c__4.5.0-9cfc7df4/lib/lwt -I /Users/fa
kenickels/.esy/3_____________________________________________________________/i/opam__s__ocamlfind-opam__c__1.8.1-ad96dfd8/lib/bytes -I /Users/fakenickels/.esy/3_____________________________________________________________/i/opam__s__psq-
opam__c__0.2.0-d409d938/lib/psq -I /Users/fakenickels/.esy/3_____________________________________________________________/i/opam__s__result-opam__c__1.5-ce40ece0/lib/result -I /Users/fakenickels/.esy/3_____________________________________
________________________/i/opam__s__seq-opam__c__base-15764575/lib/seq -I ../../packages/reason-sdl2/src -ccopt -O2 -ccopt -fno-strict-aliasing -ccopt -fwrapv -ccopt -Wall -ccopt -Wextra -ccopt -Werror -ccopt -I -ccopt . -ccopt -x -ccopt
objective-c -ccopt -g -o notification_cocoa.o notification_cocoa.c)
    notification_cocoa.c:10:5: error: 'NSUserNotification' is deprecated: first deprecated in macOS 11.0 - All NSUserNotifications API should be replaced with UserNotifications.frameworks API [-Werror,-Wdeprecated-declarations]
        NSUserNotification *notification = [[NSUserNotification alloc] autorelease];
        ^
    /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks/Foundation.framework/Headers/NSUserNotification.h:24:12: note: 'NSUserNotification' has been explicitly marked de
precated here
    @interface NSUserNotification : NSObject <NSCopying> {
               ^
    notification_cocoa.c:10:61: error: 'NSUserNotification' is deprecated: first deprecated in macOS 11.0 - All NSUserNotifications API should be replaced with UserNotifications.frameworks API [-Werror,-Wdeprecated-declarations]
        NSUserNotification *notification = [[NSUserNotification alloc] autorelease];
                                                                ^
    /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks/Foundation.framework/Headers/NSUserNotification.h:24:12: note: 'NSUserNotification' has been explicitly marked de
precated here
    @interface NSUserNotification : NSObject <NSCopying> {
               ^
    notification_cocoa.c:11:5: error: 'NSUserNotificationCenter' is deprecated: first deprecated in macOS 11.0 - All NSUserNotifications API should be replaced with UserNotifications.frameworks API [-Werror,-Wdeprecated-declarations]
        NSUserNotificationCenter *notificationCenter =
        ^
    /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks/Foundation.framework/Headers/NSUserNotification.h:118:12: note: 'NSUserNotificationCenter' has been explicitly ma
rked deprecated here
    @interface NSUserNotificationCenter : NSObject {
               ^
    notification_cocoa.c:12:35: error: 'NSUserNotificationCenter' is deprecated: first deprecated in macOS 11.0 - All NSUserNotifications API should be replaced with UserNotifications.frameworks API [-Werror,-Wdeprecated-declarations]
            [NSUserNotificationCenter defaultUserNotificationCenter];
                                      ^
    /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks/Foundation.framework/Headers/NSUserNotification.h:118:12: note: 'NSUserNotificationCenter' has been explicitly ma
rked deprecated here
    @interface NSUserNotificationCenter : NSObject {
               ^
    notification_cocoa.c:26:46: error: 'NSUserNotificationDefaultSoundName' is deprecated: first deprecated in macOS 11.0 - All NSUserNotifications API should be replaced with UserNotifications.frameworks API [-Werror,-Wdeprecated-declara
tions]
        [notification setSoundName:mute ? NULL : NSUserNotificationDefaultSoundName];
                                             ^
    /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks/Foundation.framework/Headers/NSUserNotification.h:115:36: note: 'NSUserNotificationDefaultSoundName' has been exp
licitly marked deprecated here
    FOUNDATION_EXPORT NSString * const NSUserNotificationDefaultSoundName API_DEPRECATED("All NSUserNotifications API should be replaced with UserNotifications.frameworks API", macos(10.8, 11.0)) API_UNAVAILABLE(ios, watchos, tvos);
                                       ^
    notification_cocoa.c:36:5: error: 'NSUserNotification' is deprecated: first deprecated in macOS 11.0 - All NSUserNotifications API should be replaced with UserNotifications.frameworks API [-Werror,-Wdeprecated-declarations]
        NSUserNotification *notification = [[NSUserNotification alloc] autorelease];
        ^
    /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks/Foundation.framework/Headers/NSUserNotification.h:24:12: note: 'NSUserNotification' has been explicitly marked de
precated here
    @interface NSUserNotification : NSObject <NSCopying> {
               ^
    notification_cocoa.c:36:61: error: 'NSUserNotification' is deprecated: first deprecated in macOS 11.0 - All NSUserNotifications API should be replaced with UserNotifications.frameworks API [-Werror,-Wdeprecated-declarations]
        NSUserNotification *notification = [[NSUserNotification alloc] autorelease];
                                                                ^
    /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks/Foundation.framework/Headers/NSUserNotification.h:24:12: note: 'NSUserNotification' has been explicitly marked de
precated here
    @interface NSUserNotification : NSObject <NSCopying> {
               ^
    notification_cocoa.c:37:5: error: 'NSUserNotificationCenter' is deprecated: first deprecated in macOS 11.0 - All NSUserNotifications API should be replaced with UserNotifications.frameworks API [-Werror,-Wdeprecated-declarations]
        NSUserNotificationCenter *notificationCenter =
        ^
    /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks/Foundation.framework/Headers/NSUserNotification.h:118:12: note: 'NSUserNotificationCenter' has been explicitly ma
rked deprecated here
    @interface NSUserNotificationCenter : NSObject {
               ^
    notification_cocoa.c:38:35: error: 'NSUserNotificationCenter' is deprecated: first deprecated in macOS 11.0 - All NSUserNotifications API should be replaced with UserNotifications.frameworks API [-Werror,-Wdeprecated-declarations]
            [NSUserNotificationCenter defaultUserNotificationCenter];
                                      ^
    /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks/Foundation.framework/Headers/NSUserNotification.h:118:12: note: 'NSUserNotificationCenter' has been explicitly ma
rked deprecated here
    @interface NSUserNotificationCenter : NSObject {
               ^
    notification_cocoa.c:52:46: error: 'NSUserNotificationDefaultSoundName' is deprecated: first deprecated in macOS 11.0 - All NSUserNotifications API should be replaced with UserNotifications.frameworks API [-Werror,-Wdeprecated-declara
tions]
        [notification setSoundName:mute ? NULL : NSUserNotificationDefaultSoundName];
                                                 ^
    /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks/Foundation.framework/Headers/NSUserNotification.h:115:36: note: 'NSUserNotificationDefaultSoundName' has been exp
licitly marked deprecated here
    FOUNDATION_EXPORT NSString * const NSUserNotificationDefaultSoundName API_DEPRECATED("All NSUserNotifications API should be replaced with UserNotifications.frameworks API", macos(10.8, 11.0)) API_UNAVAILABLE(ios, watchos, tvos);
                                       ^
    10 errors generated.
          ocamlc packages/reason-skia/src/wrapped/bindings/.SkiaWrappedBindings.objs/byte/skiaWrappedBindings__Skia_generated_type_stubs.{cmi,cmo,cmt}
    File "packages/reason-skia/src/wrapped/bindings/skia_generated_type_stubs.ml", line 91, characters 4-63:
    91 |   | Struct ({ tag = "sk_fontmetrics_t"} as s'), "fMaxCharWidth" ->
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    Warning 11: this match case is unused.
    File "packages/reason-skia/src/wrapped/bindings/skia_generated_type_stubs.ml", line 94, characters 4-63:
    94 |   | Struct ({ tag = "sk_fontmetrics_t"} as s'), "fAvgCharWidth" ->
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    Warning 11: this match case is unused.
    File "packages/reason-skia/src/wrapped/bindings/skia_generated_type_stubs.ml", line 155, characters 4-78:
    155 |   | Ctypes_static.Primitive Cstubs_internals.Int64_t, "RGB_V_SK_PIXELGEOMETRY" ->
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    Warning 11: this match case is unused.
        ocamlopt packages/reason-skia/src/wrapped/bindings/.SkiaWrappedBindings.objs/native/skiaWrappedBindings__Skia_generated_type_stubs.{cmx,o}
 File "packages/reason-skia/src/wrapped/bindings/skia_generated_type_stubs.ml", line 91, characters 4-63:
    91 |   | Struct ({ tag = "sk_fontmetrics_t"} as s'), "fMaxCharWidth" ->
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    Warning 11: this match case is unused.
    File "packages/reason-skia/src/wrapped/bindings/skia_generated_type_stubs.ml", line 94, characters 4-63:
    94 |   | Struct ({ tag = "sk_fontmetrics_t"} as s'), "fAvgCharWidth" ->
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    Warning 11: this match case is unused.
    File "packages/reason-skia/src/wrapped/bindings/skia_generated_type_stubs.ml", line 155, characters 4-78:
    155 |   | Ctypes_static.Primitive Cstubs_internals.Int64_t, "RGB_V_SK_PIXELGEOMETRY" ->
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    Warning 11: this match case is unused.
          ocamlc packages/reason-skia/src/wrapped/lib/.SkiaWrapped.objs/byte/skiaWrapped__Skia_generated_stubs.{cmi,cmo,cmt}
    File "packages/reason-skia/src/wrapped/lib/skia_generated_stubs.ml", lines 1620-1621, characters 2-26:
    1620 | ..Function (CI.Pointer _, Returns (CI.Primitive CI.Bool)),
    1621 |   "sk_paint_is_autohinted"...
    Warning 11: this match case is unused.
          ocamlc packages/reason-skia/src/wrapped/lib/skia_generated_stubs.o
    skia_generated_stubs.c:51:10: warning: initializing 'void *' with an expression of type 'const void *' discards qualifiers [-Wincompatible-pointer-types-discards-qualifiers]
       void* x18 = sk_data_get_data(x17);
             ^     ~~~~~~~~~~~~~~~~~~~~~
    skia_generated_stubs.c:70:10: warning: initializing 'char *' with an expression of type 'const char *' discards qualifiers [-Wincompatible-pointer-types-discards-qualifiers]
       char* x31 = sk_string_get_c_str(x30);
             ^     ~~~~~~~~~~~~~~~~~~~~~~~~
    skia_generated_stubs.c:149:17: warning: incompatible pointer types initializing 'sk_stream_t *' (aka 'struct sk_stream_t *') with an expression of type 'sk_stream_asset_t *' (aka 'struct sk_stream_asset_t *') [-Wincompatible-pointer-t
ypes]
       sk_stream_t* x85 = sk_typeface_open_stream(x83, x84);
                    ^     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    skia_generated_stubs.c:187:62: warning: passing 'char **' to parameter of type 'const char **' discards qualifiers in nested pointer types [-Wincompatible-pointer-types-discards-qualifiers]
       sk_fontmgr_match_family_style_character(x104, x105, x106, x107, x108,
                                                                 ^~~~
    /Users/fakenickels/.esy/3_____________________________________________________________/i/esy_skia-d35022e2/include/c/sk_typeface.h:49:140: note: passing argument to parameter 'bcp47' here
    SK_C_API sk_typeface_t* sk_fontmgr_match_family_style_character(sk_fontmgr_t*, const char* familyName, sk_fontstyle_t* style, const char** bcp47, int bcp47Count, int32_t character);
                                                                                                                                               ^
    skia_generated_stubs.c:1144:22: warning: initializing 'gr_glinterface_t *' (aka 'struct gr_glinterface_t *') with an expression of type 'const gr_glinterface_t *' (aka 'const struct gr_glinterface_t *') discards qualifiers [-Wincompat
ible-pointer-types-discards-qualifiers]
       gr_glinterface_t* x1042 = gr_glinterface_create_native_interface();
                         ^       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    skia_generated_stubs.c:1585:23: warning: initializing 'sk_surfaceprops_t *' (aka 'struct sk_surfaceprops_t *') with an expression of type 'const sk_surfaceprops_t *' (aka 'const struct sk_surfaceprops_t *') discards qualifiers [-Winco
mpatible-pointer-types-discards-qualifiers]
       sk_surfaceprops_t* x1430 = sk_surface_get_props(x1429);
                          ^       ~~~~~~~~~~~~~~~~~~~~~~~~~~~
    6 warnings generated.
        ocamlopt packages/reason-skia/src/wrapped/lib/.SkiaWrapped.objs/native/skiaWrapped__Skia_generated_stubs.{cmx,o}
    File "packages/reason-skia/src/wrapped/lib/skia_generated_stubs.ml", lines 1620-1621, characters 2-26:
    1620 | ..Function (CI.Pointer _, Returns (CI.Primitive CI.Bool)),
    1621 |   "sk_paint_is_autohinted"...
    Warning 11: this match case is unused.
    error: command failed: 'dune' 'build' '-p' 'reason-harfbuzz,reason-skia,reason-sdl2,Revery' (exited with 1)
    esy-build-package: exiting with errors above...

  building revery@github:revery-ui/revery#ac848c4
esy: exiting due to errors above
```

</details>

Steps to reproduce

1. git clone
2. esy install

### PR DESCRIPTION
**Pull request by fakenickels**
fixes #90 

I tried everything but upgrading Revery 😅 
It's finally working and running again on Big Sur 🎉 

image

Sweet Timber as well
image

### CODE REVIEW

File: src/AnimatedText.re [hunk 1]
```diff
@@ -13,6 +13,7 @@ module Styles = {
 let%component make = (~delay: Time.t, ~text: string, ()) => {
   let%hook (yOffset, _state, _reset) =
     Hooks.animation(
+      ~name="yOffset",
```
Comment (fakenickels) [inline src/AnimatedText.re:16]: not sure if I'm using this correctly, are they just supposed to be global ids?

Comment (glennsl) [reply src/AnimatedText.re:16]: They're just names to identify them in the logs I believe.

**Comment by glennsl** [comment]
Ci fails on formatting it seems. Could you run `esy format` and check in the changes as well?

**Comment by fakenickels** [comment]
@glennsl done!

**Review by glennsl — APPROVED** [review]
Thanks!

**Comment by fakenickels** [comment]
Not sure why but the merge build failed on Windows
https://dev.azure.com/revery-ui/revery/_build/results?buildId=6977&view=logs&j=2d2b3007-3c5c-5840-9bb0-2b1ea49925f3&t=aa4c65f3-1696-5d74-b54b-04dd48a5c31f

**Comment by glennsl** [comment]
It's just Windows. It does that sometimes :shrug: Nothing to worry about, but thanks for the heads up!

**Pull request merged**